plugin/src/main/resources/translations/en-US.yml

```diff
@@ -86,6 +86,9 @@ chat:
     successfully-completed-rewards-prefix: '    <highlight>Rewards:'
     successfully-completed-rewards-rewardformat: '    <highlight> - <unimportant>%reward%'
     successfully-completed-rewards-suffix: <EMPTY>
+    taskCompletion:
+      deliverItems:
+        delivery-of-npc-complete: '<GREEN>You have successfully delivered the NPC <highlight> %NPCNAME%'
```

Comment:
Move to taskDescription, just so objective stuff is all together in one specific place instead of spread out